contexts/arbo-context/filter-update-steps/update-pediatric-age-group-filter.tsx

```diff
@@ -0,0 +1,29 @@
+import {
+  HandleArboFilterUpdateInput,
+  HandleArboFilterUpdateOutput,
+} from "../arbo-filter-update-steps";
+
+export const updatePediatricAgeGroupFilter = (
```

Comment:
The reason why I need this is because if you deselect 'Children and Youth (0-17 years)' or select another age group as well, I have to get rid of the pediatric age group filter because even though the filter dissapears from the UI I need to remove it from our logic as well so that you don't just see no data or deselect the "Children and Youth" and be confused as to why you're still seeing only seeing data for 0-4 year olds.